package.json

```diff
@@ -74,9 +74,9 @@
     "access": "public"
   },
   "dependencies": {
-    "baseui": ">=12.2.0",
-    "react": ">=17.0.2",
-    "react-dom": ">=17.0.2"
+    "baseui": "^12.2.0",
```

Comment:
Why to we need that?
```>=``` means any version equal or bigger to the mentioned release.
```^``` will allow only versions 17.0.*